OPERATIONS.rst

```diff
@@ -0,0 +1,97 @@
+PROJECT OPERATIONS
+==================
+
+
+What is AWS SAM CLI
+-------------------
+``sam`` is the AWS CLI tool for managing Serverless applications. SAM CLI supports you through your entire 
+workflow of developing a serverless app. It allows you to quickly getting started, develop, test, and debug 
+your application locally, build, package, deploy to the cloud, and monitor and troubleshoot a deployed 
+application. It supports applications defined using AWS Serverless Application Model (SAM). 
+
+
+Core Team
+---------
+SAM CLI project is still small enough that we don't have a official team structure or governance. Therefore "Core Team" simply consists of developers who have commit access to the this Github Repository. They know most about the source code and work on it every day. 
+
+
+Contributors
+------------
+We are actively looking for contributors to join the project. You could help us fix bugs, write features, improve documentation or even do Marketing/PR activities. Read the DEVELOPMENT_GUIDE.rst file for information on how to get started. 
```

Comment:
Should this be Read the DEVELOPMENT_GUIDE or the CONTRIBUTING or both?

Reply:
Updated